### DISCUSSION TITLE
[Suggestion] Select Column should not show table name if table is specified

### DISCUSSION THREAD

**Discussion by bernatagulloesbrina** [TabularEditor/TabularEditor3#1132] [Ideas]
I see that the table name is shown for each column in the SelectColumn dialog box, even if only columns from one table are shown. That makes it harder to use 

![imatge](https://github.com/TabularEditor/TabularEditor3/assets/40455069/8895985a-66d1-4360-a955-2c336dab0479)

Actually in TE2 works fine and hides the table name. 

Regards! 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by otykier** [comment]
Good suggestion - thanks!
